manual/manual/allfiles.etex

```diff
@@ -80,15 +78,15 @@ and as a
 \input{stdlib-blurb.tex}
 \input{compilerlibs.tex}
 \input{libunix.tex}
-\input{libnum.tex}
 \input{libstr.tex}
 \input{libthreads.tex}
-\input{libgraph.tex}
+%\input{libgraph.tex}
 \input{libdynlink.tex}
-\input{libbigarray.tex}
+%\input{libbigarray.tex}
+\input{libnum.tex}
 
-\part{Appendix}
-\label{p:appendix}
+\part{Indexes}
+\label{p:indexes}
```

Comment:
The Latin pedant in me was about to shout "it's INDICES, not INDEXES!", but a quick Web search convinced me that "indexes" is fine to designate those parts of a book that list words along with page numbers.  In a more mathematical context I would still fight for "indices".

Reply:
Indeed, I was unsure too. I settled on indexes when I stumbled upon the "Society of Indexers":

https://www.indexers.org.uk/about-indexing/authors-indexes/